docs/pages/api.rst

```diff
@@ -551,6 +551,19 @@ Signature: ``Collection::count(): int;``
     $collection = Collection::fromIterable(range('a', 'c'))
         ->count(); // 3
 
+countIn
+~~~~~~~
+
+This operation requires a reference to a parameter that will contain the amount
+of items in the collection.
```

Comment:
might be nice to add a tip to show when this would be useful vs just using `count`

Reply:
Fixed